Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -279,6 +279,10 @@ extension BrowserViewController: WKNavigationDelegate {
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
         if let tab = tabManager[webView] {
             navigateInTab(tab: tab, to: navigation)
+            // Cliqz: send page view signal
+            if !tab.isPrivate && !(tab.url?.isAboutURL ?? true) {
```

Comment:
I think the default value for tab.url?.isAboutURL should be false, because the probability of a url being the AboutURL is very small. So tab.url?.isAboutURL ?? true should be tab.url?.isAboutURL ?? false

Reply:
I put it to true in case the `url` is null, because of the `url` is not null it `isAboutURL` will always return a value